go/keymanager/secrets/grpc.go

```diff
@@ -12,8 +12,10 @@ import (
 
 var (
 	// serviceName is the gRPC service name.
-	serviceName = cmnGrpc.NewServiceName("KeyManager")
+	serviceName = cmnGrpc.NewServiceName("KeyManager.Secrets")
```

Comment:
Is this too breaking (not sure who even calls this endpoint, since all nodes run consensus)?

Reply:
Should be fine.

Reply:
Should be fine. Maybe expose both for a few versions and use the new one in clients?